internal/orchestrator/subscriptions.go

```diff
@@ -47,6 +47,15 @@ func (or *orchestrator) createUpdateSubscription(ctx context.Context, subDef *co
 		return nil, i18n.NewError(ctx, coremsgs.MsgSystemTransportInternal)
 	}
 
+	if subDef.Options.TLSConfigName != "" {
```

Comment:
Is this defunct now we've moved to WS config?

Reply:
WebhookSubOptions are just inherited in Options as such  https://github.com/hyperledger/firefly/blob/main/pkg/core/subscription.go#L100 so we can reference them directly